code/modules/organs/robolimbs.dm

```diff
@@ -82,6 +82,25 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
 	parts = list(BP_HEAD)
 	monitor_styles = standard_monitor_styles
 
+/datum/robolimb/rook //Knockoff of Bishop.
+	company = "Rook"
+	desc = "This limb has a solid plastic casing with blue lights along it."
+	icon = 'icons/mob/human_races/cyberlimbs/rook/rook_main.dmi'
+	unavailable_to_build = 1
+
+/datum/robolimb/cybersolutions
+	company = "Cyber Solutions"
+	desc = "This limb is grey and rough, with little in the way of aesthetic."
+	icon = 'icons/mob/human_races/cyberlimbs/cybersolutions/cybersolutions_main.dmi'
+	unavailable_to_build = 1
+
+/datum/robolimb/cybersolutions
```

Comment:
This isn't going to work. You need to have it be a different path than any of the other datums.

Reply:
Ah, thanks.